models/migrations/v1_22/v287.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_22 //nolint
+
+import (
+	"xorm.io/xorm"
+)
+
+type HookTask struct {
+	PayloadVersion int
```

Comment:
Wouldn't it make sense to use
```suggestion
	PayloadVersion int `xorm:"DEFAULT 1"`
```
here?

Reply:
Good idea, it allows to get rid of the `update` in the migration. Done in https://github.com/go-gitea/gitea/pull/29145/commits/1e0262c9bae051de4f079b0faa3cb79f0c67b29d

Reply:
No. I don't think it will reduce the migration. The migration is still necessary.

Reply:
It's about the `engine.Update` in the migration, not the migration itself.